README.md

```diff
@@ -3,9 +3,8 @@
 *HypothesisTests.jl* is a Julia package that implements a wide range of hypothesis tests.
 
 - **Build & Testing Status:**
-  [![Build status](https://github.com/JuliaStats/HypothesisTests.jl/workflows/CI/badge.svg)](https://github.com/JuliaStats/HypothesisTests.jl/actions?query=workflow%3ACI+branch%3Amaster)
-  [![Coverage Status](https://coveralls.io/repos/JuliaStats/HypothesisTests.jl/badge.svg?branch=master)
-  ](https://coveralls.io/r/JuliaStats/HypothesisTests.jl?branch=master)
+[![Build Status](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml)
+[![Coverage Status](https://codecov.io/gh/JuliaStats/HypothesisTests.jl/branch/master/graph/badge.svg?token=ztSoTXYVhb)](https://codecov.io/gh/JuliaStats/HypothesisTests.jl)
 
 - **Documentation**: [![][docs-stable-img]][docs-stable-url] [![][docs-latest-img]][docs-latest-url]
```

Comment:
One additional change: `latest` was removed and `dev` does not link to it anymore (https://github.com/JuliaDocs/Documenter.jl/blob/master/CHANGELOG.md#version-v0240) so I think `docs-latest-url` should be changed to http://JuliaStats.github.io/HypothesisTests.jl/dev/ (and you might want to replace `latest` with `dev` also in the badge).